app/controllers/v1/tasks_controller.rb

```diff
@@ -31,7 +31,7 @@ def no_business
     if task.completed?
       render jsonapi: task.submissions.first
     else
-      task.file_no_business!
+      task.file_no_business!(current_user)
```

Comment:
As above, an update to the request spec (`spec/requests/v1/tasks_spec.rb`) to verify the user is actually recorded against the submission would be good.